internal/store/common.go

```diff
@@ -297,3 +299,158 @@ func retryUpdate(fn func() (bool, error)) error {
 	}
 	return err
 }
+
+func configsAreEqual(c1, c2 *[]api.DeviceSpec_Config_Item) bool {
```

Comment:
Maybe we can use this generic func https://pkg.go.dev/slices#EqualFunc as an envelope for all the functions?

Reply:
Nice.  Done.